pkg/crc/version/version.go

```diff
@@ -29,7 +29,7 @@ var (
 )
 
 const (
-	releaseInfoLink = "https://mirror.openshift.com/pub/openshift-v4/clients/crc/latest/release-info.json"
+	releaseInfoLink = "https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/1.26.0/release-info.json"
```

Comment:
This has to be a `latest` link, not `1.26.0`, otherwise we won't be able to detect the newer version